bluebutton-data-model-rif/src/main/resources/db/scripts/Beneficiaries_with_duplicate_hicns.sql

```diff
@@ -0,0 +1,108 @@
+/*
+ * Queries for finding HICN and HICN hash collisions between beneficiary records.
+ *
+ * These queries are intended to be run manually, by copy-pasting them into an
+ * interactive `psql` session, or somesuch.
+ */
+
+-- How many "hicn"s are associated with more than one "beneficiaryId", whether or not those
+-- "beneficiaryId"s resolve to an actual "Beneficiaries" record?
+--
+-- On 2019-06-07, this was run in PROD:
+-- * returned 2,697,057 rows
+-- * with a max count of 4
+-- * in 1478941.350 ms (0:25h)
+WITH
+  hicn_id_tuples AS (
+    SELECT
+        DISTINCT "beneficiaryId" AS bene_id, "hicn" AS hicn
+      FROM "Beneficiaries"
+    UNION ALL
```

Comment:
Got a phone review from @dho937 that pointed out a major bug here: using `UNION ALL` leads to duplicates and other weirdness:

* Consider this data:
    
    ```
    1, 1A
    2, 2A
    UNION ALL
    1, 1A
    2, 2A
    3, 3A
    4, 1A
    ```
    
* Instead of getting an actual summed-`count(...)` of all the distinct-and-duplicate tuples, I'll get a count of _rows_ with each HICN, i.e. a count of 3 for the HICN "1A", instead of the correct count of 2.

This could be fixed either by switching to a `UNION` (which will de-dupe the CTE) or by switching all of the `count(bene_id)` expressions to `count(DISTINCT bene_id)`, instead.